src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -27,7 +27,7 @@ def _gen_tags(cls, all_endpoints):
                 if endpoint.is_api or all_endpoints:
                     list_of_ctrl.add(endpoint.ctrl)
 
-        TAG_MAP = {}
+        TAG_MAP = {}  # noqa: N806
```

Comment:
That's a case where `CONSTANT_CASE` is used, but the variable is actually not a constant.
```suggestion
        tag_map = {}
```